CHANGELOG.md

```
@@ -1,6 +1,17 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+### [Version 7.8.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/7.8.0) (July 27, 2026)
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)